src/main/java/org/opensearch/knn/index/quantizationService/KNNVectorQuantizationTrainingRequest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.quantizationService;
```

Comment:
this should be `org.opensearch.knn.index.quantization;`

Reply:
It's part of Client not Quantization Framework , I would keep it on Quantization service.

Reply:
If you want to keep service then atleast make it all small case or quantization.service.

Camel case in package name I have not seen anywhere that was the whole point

Reply:
Sure , I will fix this in followup PR.